src/server/command/worldedit/copy.zig

```diff
@@ -0,0 +1,44 @@
+const std = @import("std");
+
+const main = @import("root");
+const User = main.server.User;
+
+const Block = main.blocks.Block;
+const Blueprint = main.blueprint.Blueprint;
+
+pub const description = "Copy selection to clipboard.";
+pub const usage = "/copy";
+
+pub fn execute(args: []const u8, source: *User) void {
+	if(args.len != 0) {
+		source.sendMessage("#ff0000Too many arguments for command /copy. Expected no arguments.", .{});
+		return;
+	}
+	source.mutex.lock();
```

Comment:
Same here, the lock should only be taken in the success branch. And the old clipboard should only be discarded in the success branch

Reply:
How do you like the Result structure pattern? Yoinked it from Rust.

Reply:
> How do you like the Result union/enum pattern (capture)? Yoinked it from Rust.

I  don't particularly like it, but I don't really see a better way to implement it either.